### PR TITLE
Add option to disable network bridging to the parent docker network

### PR DESCRIPTION
**Pull request by karlkfi**
- Bridge disabled by default, enabled by DOCKER_NETWORK_BRIDGE=true

### CODE REVIEW

File: wrapdocker [hunk 1]
```diff
@@ -136,53 +136,57 @@ if [ "${STORAGE_FS}" != "overlay" -o "${OVERLAY_WORKS}" == false ]; then
...
-
-IP_PLUS_OFFSET=$(printf "%d.%d.%d.%d\n" \
-  "$(( n1 + ((i1 - n1 + o1) & w1) ))" \
-  "$(( n2 + ((i2 - n2 + o2) & w2) ))" \
-  "$(( n3 + ((i3 - n3 + o3) & w3) ))" \
-  "$(( n4 + ((i4 - n4 + o4) & w4) ))")
-
-FIXED_CIDR=$(ipcalc $IP_PLUS_OFFSET/$DOCKER_NETWORK_SIZE | grep Network | awk '{print $2;}')
-echo "Using network $FIXED_CIDR for docker containers"
+if [ "${DOCKER_NETWORK_BRIDGE:-false}" == "true" ]; then
```
Comment (sttts) [inline wrapdocker]: ROUTABLE_CONTAINER_IPS would be a better name. There is always a docker bridge. The only difference here is that we set it up in a special way.

**Comment by sttts** [comment]
Only the variable name is suboptimal. Otherwise lgtm

File: wrapdocker (hunk 1 above)
Comment (karlkfi) [inline wrapdocker]: I was tyring to keep it consistent with the other 2 related variables: DOCKER_NETWORK_SIZE & DOCKER_NETWORK_OFFSET. Ideally, I'd prefer not to have boolean feature toggles, but the other two are currently optional.

What about just checking `[ ! -z "${DOCKER_NETWORK_OFFSET}" ]` and making that required for enablement? We don't have any significant use cases where you only need one mesos-slave-dind running...

Comment (sttts) [inline wrapdocker]: Check for DOCKER_NETWORK_OFFSET is also fine. Just document it and/or print out that routable container IPs are disabled.

**Comment by karlkfi** [comment]
Switched to just checking for DOCKER_NETWORK_OFFSET, like we agreed. 
Also made some doc changes.

**Comment by sttts** [comment]
lgtm

**Pull request merged**